docs/user/dql/functions.rst

```diff
@@ -323,16 +323,178 @@ Specifications:
 1. DATE_FORMAT(DATE, STRING) -> STRING
 2. DATE_FORMAT(DATE, STRING, STRING) -> STRING
 
+DATE_ADD
+=====
+
+Description
+-----------
+
+Usage: date_add(date, INTERVAL expr unit) adds the time interval expr to date
+
+Argument type: DATE/DATETIME/TIMESTAMP, INTERVAL
+
+Return type:
+
+(DATE, DATE INTERVAL) -> DATE
+(DATE, TIME INTERVAL) -> DATETIME
+(DATETIME/TIMESTAMP, INTERVAL) -> DATETIME
+
+Synonyms: ADDDATE
```

Comment:
do we support ADDDATE? if not, remove this.

Reply:
we support ADDDATE